models/classes/creator/ListItemLookup.php

```diff
@@ -73,6 +71,10 @@ static function (array $node): string {
             $result['nodes']
         );
 
+        if (empty($result['nodes'])) {
+            return $result;
+        }
+
```

Comment:
This is better moved above the `array_map` call.

Reply:
it is true